### DISCUSSION TITLE
Améliorer la visualisation des fichiers audio

### DISCUSSION THREAD

**Discussion by ianme** [EsupPortail/Esup-Pod#546] [Ideas]
Ce serait intéressant d'avoir une visualisation différente pour les fichiers audio :

- Proposer a minima un player moins haut par défaut quand on embed dans une page web.
- Dans l'idéal, générer et afficher le déroulement du son sur une waveform, un peu comme fait SoundCloud.

**Comment by ptitloup** [comment]
Bonjour,
savez vous s'il existe des plugins pour videojs permettant de faire cela ?
Par curiosite, avez vous une instance de Pod en prod ? si oui, quel établissement ?
Merci

**Reply by ianme** [reply]
Bonjour,

Désolé pour le retard !

Merci pour la piste, je pense que [videojs-wavesurfer](https://www.npmjs.com/package/videojs-wavesurfer) est ce que je cherche.

J'utilise le [Pod](https://pod.univ-cotedazur.fr/) de l'Université Côte d'Azur et le besoin d'y héberger des podcasts monte en puissance (je suis en contact avec l'équipe qui gère le Pod).

Bien cordialement,
Iannis

**Reply by ptitloup** [reply]
Bonjour, 
merci pour votre retour. 
Si vous implémentez ce plugin dans Pod, merci de nous faire un retour pour qu'on puisse l'intégrer pour tout le monde. (L'université Côte d'Azur contribue déjà au projet Pod)
Bien cordialement

**Reply by Badatos** [reply]
Merci @ianme pour cette proposition. videojs-wavesurfer sera inclus dans la prochaine mise à jour majeure de Pod (3.0.0)

**Reply by ianme** [reply]
C'est parfait @Badatos un grand merci pour ta réactivité et ton travail !
J'ai hâte de découvrir toutes les nouveautes de la v3 alors !
